azure-pipelines.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     displayName: WebJobs extension
     steps:
       - powershell: | # Allow tags matching v1.2.3 and v1.2.3-xyz1
-          $found = '$(Build.SourceBranchName)' | Select-String -Pattern '^v((\d+\.\d+\.\d+)(?:-\w+))*$'
+          $found = '$(Build.SourceBranchName)' | Select-String -Pattern '^v((?:0|[1-9]\d*)\.(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)(?:-\w+)?)$'
           if (-not $found) {
```

Comment:
The earlier pattern was incorrect (don't remember why I even committed it like this 🤦). It did not allow a tag like `v2.0.0` which is intended tag for next major release.

Reply:
I should have also look at it it carefully the previous time, my bad.

Reply:
I don't think you reviewed the PR with this particular change :)

Reply:
I had added this PR comment after the I added the first commit. I should have checked the regex when I had worked on build pipeline 1-2 months back.